azext_iot/sdk/iothub/controlplane/models/_models_py3.py

```diff
@@ -0,0 +1,4499 @@
+     Required.
+    :vartype description: str
+    :ivar actions_required: Actions required for a private endpoint connection.
+    :vartype actions_required: str
+    """
+
+    _validation = {
+        'status': {'required': True},
+        'description': {'required': True},
+    }
+
+    _attribute_map = {
+        "status": {"key": "status", "type": "str"},
+        "description": {"key": "description", "type": "str"},
+        "actions_required": {"key": "actionsRequired", "type": "str"},
+    }
+
+    def __init__(
+        self,
+        *,
+        status: Union[str, "_models.PrivateLinkServiceConnectionStatus"],
+        description: str,
+        actions_required: Optional[str] = None,
+        **kwargs
+    ):
+        """
+        :keyword status: The status of a private endpoint connection. Required. Known values are:
+         "Pending", "Approved", "Rejected", and "Disconnected".
+        :paramtype status: str or
+         ~azure.mgmt.iothub.v2022_04_30_preview.models.PrivateLinkServiceConnectionStatus
+        :keyword description: The description for the current state of a private endpoint connection.
+         Required.
+        :paramtype description: str
+        :keyword actions_required: Actions required for a private endpoint connection.
+        :paramtype actions_required: str
+        """
+        super().__init__(**kwargs)
+        self.status = status
+        self.description = description
+        self.actions_required = actions_required
+
+
+class RegistryStatistics(msrest.serialization.Model):
+    """Identity registry statistics.
+
+    Variables are only populated by the server, and will be ignored when sending a request.
+
+    :ivar total_device_count: The total count of devices in the identity registry.
+    :vartype total_device_count: int
+    :ivar enabled_device_count: The count of enabled devices in the identity registry.
+    :vartype enabled_device_count: int
+    :ivar disabled_device_count: The count of disabled devices in the identity registry.
+    :vartype disabled_device_count: int
+    """
+
+    _validation = {
+        'total_device_count': {'readonly': True},
+        'enabled_device_count': {'readonly': True},
+        'disabled_device_count': {'readonly': True},
+    }
+
+    _attribute_map = {
+        "total_device_count": {"key": "totalDeviceCount", "type": "int"},
+        "enabled_device_count": {"key": "enabledDeviceCount", "type": "int"},
+        "disabled_device_count": {"key": "disabledDeviceCount", "type": "int"},
+    }
+
+    def __init__(
+        self,
+        **kwargs
+    ):
+        """
+        """
+        super().__init__(**kwargs)
+        self.total_device_count = None
+        self.enabled_device_count = None
+        self.disabled_device_count = None
+
+
+class RootCertificateProperties(msrest.serialization.Model):
+    """This property store root certificate related information.
+
+    @vilit - changed the type of the last_updated_rot_certificate_v2 to a rfc-1123
```

Comment:
The change was made to last_updated_time_utc/lastUpdatedTimeUtc (a datetime attribute).